src/ConstVisualizer/ConstFinder.cs

```diff
@@ -308,18 +358,47 @@ public static string GetQualification(CSharpSyntaxNode dec)
             return result.TrimEnd('.');
         }
 
+        public static string GetQualificationVisualBasic(VBasicCodeAnalysis.VisualBasicSyntaxNode dec)
+        {
+            string result = string.Empty;
+            SyntaxNode parent = dec.Parent;
+
+            while (parent != null)
+            {
+                if (parent is VBasicSyntax.TypeBlockSyntax tbs)
+                {
+                    result = $"{tbs.BlockStatement.Identifier.ValueText}.{result}";
+                    parent = tbs.Parent;
+                }
+                else if (parent is VBasicSyntax.NamespaceBlockSyntax nbs)
+                {
+                    result = $"{nbs.NamespaceStatement.Name}.{result}";
+                    parent = nbs.Parent;
+                }
+                else
+                {
+                    parent = parent.Parent;
+                }
+            }
+
+            return result.TrimEnd('.');
+        }
+
         public static bool IsConst(SyntaxNode node)
         {
-            return node.ChildTokens().Any(t => t.IsKind(SyntaxKind.ConstKeyword));
+            return node.ChildTokens().Any(t => t.IsKind(CsharpCodeAnalysis.SyntaxKind.ConstKeyword) ||
+                                               t.IsKind(VBasicCodeAnalysis.SyntaxKind.ConstKeyword));
         }
 
         internal static async Task<EnvDTE.Document> SafeGetActiveDocumentAsync(EnvDTE.DTE dte)
         {
+            await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
             try
             {
                 // Some document types (inc. .csproj) throw an error when try and get the ActiveDocument
                 // "The parameter is incorrect. (Exception from HRESULT: 0x80070057 (E_INVALIDARG))"
-                return await Task.FromResult(dte?.ActiveDocument);
+                EnvDTE.Document doc = await Task.FromResult(dte?.ActiveDocument);
+                return doc != null && (doc.Language == "CSharp" || doc.Language == "Basic") ? doc : null;
```

Comment:
Why is this language check needed? 
How do you even get this code to execute to this line if not in a supported language?